var/spack/repos/builtin/packages/libnetworkit/package.py

```diff
@@ -34,7 +35,8 @@ class Libnetworkit(CMakePackage):
     depends_on('libtlx')
     depends_on('py-sphinx', when='+doc', type='build')
 
-    patch('0001-Name-agnostic-import-of-tlx-library.patch', when='@6.1:')
+    patch('0001-Name-agnostic-import-of-tlx-library.patch', when='@6.1:8.1')
+    patch('0001-Name-agnostic-import-of-tlx-library-90.patch', when='@9.0:')
```

Comment:
Has this patch been contributed upstream? I would love to hear the developers opinion on it to make sure it's a good idea, as I'm not familiar with the project myself.

Reply:
In principal it does the trick. 

However we decided not to use it upstream, since the same could be achieved by using pkgconfig for finding the correct lib-name/version. It is still on my agenda to test this properly for every possible environment. Once this is integrated upstream, I will remove the patch here of course.